mcodingbot/plugins/reactions.py

```diff
@@ -0,0 +1,25 @@
+import re
+
+import crescent
+import hikari
+
+RUST_REGEX = re.compile(r"\brust\b", flags=re.I)
+
+plugin = crescent.Plugin("reactions")
+
+
+@plugin.include
+@crescent.event
+async def on_message(event: hikari.MessageCreateEvent) -> None:
```

Comment:
hikari.GuildMessageCreateEvent?

Reply:
Any reason for it to not work in DMs? Other commands do.

Reply:
This isn't a command though? I guess it's ok for Rust :rocket: to work in DMs.

I'm fine either way.

Reply:
the ping command works in DMs

Reply:
Are you sure? Pretty sure discord added `dm_permissions` which is False by default.

Reply:
Nope you're right, nevermind lol.